index.js

```diff
@@ -25,47 +49,71 @@ function makeCopier(copyConfig, assetManager, { compact } = {}) {
 		filter: copyConfig.filter
 	});
 	let { fingerprint } = copyConfig;
-	let plugins = determinePlugins(compact, copyConfig);
+	let compactors = determineCompactors(compact, copyConfig);
 
 	return filepaths => {
 		return Promise.all([
 			(filepaths ? fileFinder.match(filepaths) : fileFinder.all()),
 			determineTargetDir(source, target)
 		]).then(([fileNames, targetDir]) => {
```

Comment:
Aside: This would be slightly more elegant with `await` - in which case you could also inline `processFiles`?

Reply:
Yes, good point, solved in b14042e6a5f7a9ff1159dcae456fb9bc6eb57e48